Template.php

```diff
@@ -2463,6 +2464,7 @@ function assessUrl($url){
         curl_setopt($ch, CURLOPT_NOBODY, 1);
         curl_setopt($ch, CURLOPT_HEADER, FALSE);
         curl_exec($ch);
+        curl_setopt($ch, CURLOPT_VERBOSE, 0); // stop curl from printing non-errors to standard out
```

Comment:
Presumably this ought to come before the `curl_exec` on the previous line?

Reply:
Ooops.  Fixed.